### PR TITLE
GooglePlayBooks: Modified ActionLogger tags and capture points

### PR DESCRIPTION
**Pull request by mcgeagh**

### CODE REVIEW

File: wlauto/workloads/googleplaybooks/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 1]
```diff
@@ -307,9 +311,6 @@ private void gesturesTest() throws Exception {
                 case UIDEVICE_SWIPE:
                     uiDeviceSwipe(dir, steps);
                     break;
-                case UIOBJECT_SWIPE:
-                    uiObjectSwipe(pageView, dir, steps);
-                    break;
```
Comment (ep1cman) [inline wlauto/workloads/googleplaybooks/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java]: Why was this removed?

Comment (mcgeagh) [inline wlauto/workloads/googleplaybooks/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java]: Cody Tidy. Case is never hit.
